sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -377,7 +421,170 @@
+          }
+        },
+        "RequiresDuplicateDetection": {
+          "description": "A value indicating if this queue requires duplicate detection.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "RequiresSession": {
+          "description": "A value that indicates whether the queue supports the concept of sessions.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "Status": {
+          "$ref": "#/definitions/EntityStatus",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "SupportOrdering": {
+          "description": "A value that indicates whether the queue supports ordering.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        }
+      }
+    },
+    "ServiceBusManagementError": {
+      "description": "The error response from Service Bus.",
+      "type": "object",
+      "properties": {
+        "Code": {
+          "description": "The service error code.",
+          "type": "integer"
+        },
+        "Detail": {
+          "description": "The service error message.",
+          "type": "string"
+        }
+      }
+    },
+    "TopicProperties": {
+      "description": "Description of a Service Bus topic resource.",
+      "type": "object",
+      "xml": {
+        "name": "TopicDescription",
+        "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+      },
+      "properties": {
+        "TopicName": {
+          "description": "Name of the topic",
+          "type": "string"
+        },
+        "AuthorizationRules": {
+          "description": "Authorization rules for resource.",
+          "type": "array",
+          "xml": {
+            "wrapped": true,
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          },
+          "items": {
+            "$ref": "#/definitions/AuthorizationRule",
+            "xml": {
+              "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+            }
+          }
+        },
+        "AutoDeleteOnIdle": {
+          "description": "ISO 8601 timeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes.",
+          "type": "string",
+          "format": "iso8601",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "CreatedAt": {
+          "description": "The exact time the queue was created.",
+          "type": "string",
```

Comment:
`"format": "date-time"` for date time type is required so the generated model class doesn't use plain string.

Reply:
This was a misadd. The property exists in "TopicRuntimeInfo"